pkg/machine/ignition/ignition_linux.go

```diff
@@ -1,29 +1,29 @@
 package ignition
 
 import (
+	"errors"
 	"os"
-	"os/exec"
+	"path/filepath"
 	"strings"
-
-	"github.com/sirupsen/logrus"
 )
 
 func getLocalTimeZone() (string, error) {
-	trimTzFunc := func(s string) string {
-		return strings.TrimPrefix(strings.TrimSuffix(s, "\n"), "Timezone=")
+	path, err := filepath.EvalSymlinks("/etc/localtime")
```

Comment:
if we are going to do this, I think it is generally accepted practice to honor $TZ environment variables.  WDYT?

Reply:
Good idea, I guess we would like to use TZ for all platforms and not just linux then?